packages/stark-ui/src/modules/app-logout/components/app-logout.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Inject, Input, OnInit, ViewEncapsulation } from "@angular/core";
+
+import {
+	STARK_LOGGING_SERVICE,
+	STARK_ROUTING_SERVICE,
+	STARK_APP_CONFIG,
+	StarkLoggingService,
+	StarkRoutingService,
+	StarkApplicationConfig,
+	STARK_SESSION_SERVICE,
+	StarkSessionService
+} from "@nationalbankbelgium/stark-core";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-app-logout";
+
+/**
+ * Component to display the application's logout button
+ */
+@Component({
+	selector: "stark-app-logout",
+	templateUrl: "./app-logout.component.html",
+	styleUrls: ["./app-logout.component.scss"],
```

Comment:
For all our components we don't include the SCSS files via the component's styleUrls. Why? Because we cant to give the developers the freedom to use our styles or not, so they should imported them manually in their application (in our case in the showcase app)